yb-voyager/src/dbzm/dbzm.go

```diff
@@ -64,12 +75,111 @@ func findDebeziumDistribution() error {
 	return nil
 }
 
+func GenerateAndStoreYBStreamID(config *Config) (string, error) {
+	err := findDebeziumDistribution(config.SourceDBType)
+	if err != nil {
+		return "", fmt.Errorf("error in finding debezium distribution: %s", err)
+	}
+	YB_CLIENT_WRAPPER_JAR := fmt.Sprintf("%s/yb-client-cdc-stream-wrapper.jar", DEBEZIUM_DIST_DIR) //$DEBEZIUM_DIST/yb-client-cdc-stream-wrapper.jar
+	tableName := strings.Split(config.TableList[0], ".")[1]                                        //any table name in the database is required by yb-client createCDCStream(...) API
+	command := fmt.Sprintf("java -jar %s -create -master_addresses %s -table_name %s -db_name %s ", YB_CLIENT_WRAPPER_JAR, config.YBServers, tableName, config.DatabaseName)
+
+	if config.SSLRootCert != "" {
+		command += fmt.Sprintf(" -ssl_cert_file %s", config.SSLRootCert)
+	}
+
+	cmd := exec.CommandContext(context.Background(), "/bin/bash", "-c", command)
+	var outbuf bytes.Buffer
+	var errbuf bytes.Buffer
+	cmd.Stdout = &outbuf
+	cmd.Stderr = &errbuf
+	err = cmd.Start()
+	if err != nil {
+		if outbuf.String() != "" {
+			log.Infof("Output of the command %s: %s", command, outbuf.String())
+		}
+		log.Infof("Failed to start command: %s, error: %s", command, err)
+		return "", err
+	}
+	err = cmd.Wait()
+	if err != nil {
+		if outbuf.String() != "" {
+			log.Infof("Output of the command %s: %s", command, outbuf.String())
+		}
+		log.Infof("Failed to wait for command: %s , error: %s", command, err)
+		return "", err
+	}
+	//output of yb-admin command - CDC Stream ID: <stream_id>
+	output := outbuf.String()
+	streamID := strings.Trim(strings.Split(output, ":")[1], " \n")
+	//save streamID in a file
+	streamIDFile := filepath.Join(config.ExportDir, "metainfo", "yb_cdc_stream_id.txt")
+	file, err := os.Create(streamIDFile)
+	if err != nil {
+		return "", fmt.Errorf(" creating file: %s, error: %s", streamIDFile, err)
```

Comment:
Avoid a white space after the opening double-quote `" creating..."`. Fix all such cases.